pkg/svcat/service-catalog/class_test.go

```diff
@@ -207,4 +207,38 @@ var _ = Describe("Class", func() {
 			Expect(actions[0].(testing.GetActionImpl).Name).To(Equal(fakeClassName))
 		})
 	})
+	Describe("CreateClassByName", func() {
```

Comment:
I am not sure if this is the right name for the test, no? The function takes in the entire class resource, not just a name. How about `CreateClass` instead?